### PR TITLE
[Core] Test to fix #7067

### LINKED ISSUE #7067: windows CI fails
The windows CI seems to be failing for many recent PRs
https://github.com/KratosMultiphysics/Kratos/pull/7045/checks?check_run_id=763838897#step:6:493

https://github.com/KratosMultiphysics/Kratos/pull/7066/checks?check_run_id=762126529#step:6:493

https://github.com/KratosMultiphysics/Kratos/pull/7064/checks?check_run_id=762248826#step:6:491

for example. Calling code doctors @philbucher @roigcarlo 🏥

### PR DESCRIPTION
**Pull request by loumalouomega**
Fixes #7067 ?

Just testing

### CODE REVIEW

**Comment by loumalouomega** [comment]
Now it works..., lets see

**Comment by loumalouomega** [comment]
@philbucher @roigcarlo looks like this fixes #7067

**Comment by loumalouomega** [comment]
Still needs some fixes in order to work in linux

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -245,6 +245,7 @@ jobs:
           -DBOOST_ROOT="%BOOST_ROOT_1_72_0%"                 ^
           -DINSTALL_RUNKRATOS=OFF                            ^
           -DCMAKE_CXX_FLAGS="/Od /we4661 /we4804"            ^
+          -DLOCAL_ZLIB_COMPILATION=ON                        ^
```
Comment (philbucher) [inline .github/workflows/ci.yml]:
```suggestion
          -DFORCE_LOCAL_ZLIB_COMPILATION=ON                        ^
```

can you please also add this in the nightly build?

File: CMakeLists.txt [hunk 2]
```diff
@@ -359,26 +359,29 @@ if(${USE_MPI} MATCHES ON )
 endif(${USE_MPI} MATCHES ON )
 
 # If we do not locally compile ZLIB we will find ZLIB in the system, if we cannot find it we will enforce the local compilation
-if( LOCAL_ZLIB_COMPILATION MATCHES OFF )
+IF(NOT DEFINED FORCE_LOCAL_ZLIB_COMPILATION)
+    set( FORCE_LOCAL_ZLIB_COMPILATION OFF )
+ENDIF(NOT DEFINED FORCE_LOCAL_ZLIB_COMPILATION)
```
Comment (philbucher) [inline CMakeLists.txt]: [check this out](https://cmake.org/cmake/help/v3.13/command/option.html):
```suggestion
OPTION ( FORCE_LOCAL_ZLIB_COMPILATION "force local compilation of zlib" OFF )
```

@roigcarlo any reason we don't use this more often?

Comment (loumalouomega) [reply CMakeLists.txt]: What does that black magic you are showing me?

Comment (loumalouomega) [reply CMakeLists.txt]: I see, the link explains, hail to the link

**Review by philbucher — APPROVED** [review]
thx
looks good to me, we can still change later if needed :)

**Pull request merged**